src/org/isf/admission/test/Tests.java

```diff
@@ -2,11 +2,14 @@
 
 
 import static org.junit.Assert.assertEquals;
```

Comment:
(this reminds me we should separate source from test, which we'll do in [OP-181](https://openhospital.atlassian.net/browse/OP-181) - see also https://github.com/informatici/openhospital-core/pull/47 )

Reply:
Yeah, that should land on core soon! ;)